### DISCUSSION TITLE
下载茉莉花插件后，知网抓取论文没有反应

### DISCUSSION THREAD

**Discussion by Oceanank** [l0o0/jasminum#277] [Q&A]
目前使用的是zotero7，茉莉花插件为1.0.0-16版本，在知网抓取论文时，虽然已经显示了条目信息，但未出现PDF，也没有出现抓取失败的提示，而是没有反映，已排除引起该问题的常见问题，但是还是没有反应。请问这种情况应该如何解决？
<img width="316" alt="bef336120eb55d0971c073a9dc62f81" src="https://github.com/l0o0/jasminum/assets/170303549/26455253-da5d-40d1-bebd-72fd350d0938">

**Comment by Dule233** [comment]
你解决了吗，我也是这样，排除了所有常见问题，还是一样

**Comment by jiaojiaodubai** [comment]
1. 茉莉花不参与浏览器的抓取过程
2. 浏览器使用 Connector 调用 Translator进行抓取
3. 茉莉花用于下载中文 Translator
4. 先[阅读教程](https://zotero-chinese.com/user-guide/faqs/update-translators)进行基本排查
5. VPN 环境下附件抓取不稳定请尝试使用其他认证登录方式
6. 浏览器抓取问题请到[Translators_CN](https://github.com/l0o0/translators_CN)反馈
